src/adagrams.js

```diff
@@ -1,8 +1,164 @@
 const Adagrams = {
+  allLetters: {
+    A: 9,
+    B: 2,
+    C: 2,
+    D: 4,
+    E: 12,
+    F: 2,
+    G: 3,
+    H: 2,
+    I: 9,
+    J: 1,
+    K: 1,
+    L: 4,
+    M: 2,
+    N: 6,
+    O: 8,
+    P: 2,
+    Q: 1,
+    R: 6,
+    S: 4,
+    T: 6,
+    U: 4,
+    V: 2,
+    W: 2,
+    X: 1,
+    Y: 2,
+    Z: 1
+  },
   drawLetters() {
-    // Implement this method for wave 1
+    let letterPool = [];
+
+    Object.keys(this.allLetters).forEach(letter => {
+
+      let times = this.allLetters[letter];
+      for (let i = 0; i < times; i += 1) {
+        letterPool.push(letter);
+      }
+    });
+    letterPool = this.shuffle(letterPool);
+    const hand = letterPool.slice(0, 10);
+    // console.log(hand);
+    return hand;
+  },
+  shuffle(array) {
+    let currentIndex = array.length;
+    let temporaryValue;
+    let randomIndex;
```

Comment:
Unless you came up with this shuffle algorithm yourself, you should include a source for things like this.

Something like this:

```js
// Source: https://stackoverflow.com/a/26165799/1513338
```